packages/components/FocusZone/src/FocusZone.types.ts

```diff
@@ -45,6 +67,7 @@ export type FocusZoneProps = React.PropsWithChildren<{
 
 export interface NativeProps extends Omit<FocusZoneProps, 'isCircularNavigation'> {
```

Comment:
driveby nit, should `isCircularNavigation` be deprecated?

Reply:
I don't believe so, it's still a heavily used abstraction for arrow keys. What's your thinking here?

Reply:
Will continue this offline